docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -0,0 +1,173 @@
+
+### Use cases
+
+Moderated Sessions are useful in the following scenarios:
+- When you have stringent security requirements and need to have people watching over user-initiated sessions to a set of servers.
+- When you want to share a terminal with someone else to be able to instruct or collaborate.
+
+## Policies
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and who is required to be present to start one.
+
+The system is based around **require policies** and **allow policies**. 
+
+Require policies define a set of conditions that must be a met for a session to start or run.
+A minimum of one policy from each relevant role the user has must match for the session to start.
+
+Allow policies are used to define what sessions a user can join
+and under what conditions they may join a session.
+
+## Configuring Moderated Sessions
+
+### `require_session_join`
+
+#### Options
+
+The following are required options for `require_session_join`:
+
+|Option|Type|Description|
+|---|---|---|
+|`name`|String|The name of the require policy|
+|`filter`|[Filter](./moderated-sessions.mdx#filters)|An expression that, if it evaluates to true for a given user, enables the user to be present in a Moderated Session|
+|`kinds`|`[]`[Session kind](./moderated-sessions.mdx/#session-kinds)|The kind of session that the policy applies to|
+|`modes`|`[]`[Participant mode](./moderated-sessions.mdx#participant-modes)|The participant mode that applies to the user joining the Moderated Session under this policy|
+|`count`|[Participant mode](./moderated-sessions.mdx#require-policy-count)|The amount of users that need to match the filter expression to satisfy the policy|
+
+#### Example
+
+The policy below specifies that a given role
+must have a minimum of two users with the role `auditor` and the mode `moderator` present in the session
+to start it. The policy applies to SSH and Kubernetes sessions only.
+
+When a user with this require policy starts a session, it will be pending
+until the policy is fulfilled. 
+
+```yaml
+kind: role
+metadata:
+  name: prod-access
+spec:
+  allow:
+    require_session_join:
+      - name: Auditor oversight
+        filter: 'contains(user.roles, "auditor")'
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator']
+        count: 2
+```
+
+### `join_sessions`
+
+#### Options
+
+The following are required options for `join_sessions`:
+
+|Option|Type|Description|
+|---|---|---|
+|`name`|String|The name of the allow policy|
+|`roles`|[]String|A list of names for Teleport roles that this policy applies to. Users with this role are eligible to join a Moderated Session under this policy.|
+|`kinds`|`[]`[Session kind](./moderated-sessions.mdx#session-kinds)|The kind of session that the policy applies to|
+|`modes`|`[]`[Participant mode](./moderated-sessions.mdx#participant-modes)|The participant mode that paplies to the user joining the Moderated Session under this policy|
```

Comment:
A suggestion was made by @ptgott here to allow for filter expressions instead of a roles list if I read the comment in the conceptual guide PR correctly. I reasoned this in the original RFD and ended up sticking with the roles list to make the intended way to construct access hierarchies/systems using these policies more explicit. I feel like a filter expression gives flexibility that doesn't give us any better capabilities in return.